scalafix-nsc/src/test/resources/checkSyntax/OrganizeImportsNoGroups.source

```diff
@@ -0,0 +1,28 @@
+rewrites = []
+imports.organize = true
+imports.alwaysUsed = []
```

Comment:
I just did, and also verified that the test still triggers the bug. Note however, that there is also a downside on relying heavily on default values in tests:

1) a reader of the test has to consult them, to see what is actually tested
2) defaults might change in the future